services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -45,12 +75,60 @@ async def list_study_jobs(
     "/{study_id:uuid}/jobs",
     response_model=Job,
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
-    status_code=status.HTTP_501_NOT_IMPLEMENTED,
-    response_description="Not implemented",
 )
-async def create_study_job(study_id: StudyID):
-    msg = f"create study job study_id={study_id!r}. SEE https://github.com/ITISFoundation/osparc-simcore/issues/4177"
-    raise NotImplementedError(msg)
+async def create_study_job(
+    study_id: StudyID,
+    job_inputs: JobInputs,
+    webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
+) -> Job:
+    try:
+        project = await webserver_api.clone_project(project_id=study_id, hidden=True)
+        job = create_job_from_study(
+            study_key=study_id, project=project, job_inputs=job_inputs
+        )
+        project = await webserver_api.update_project(
+            project_id=job.id, update_params=ProjectUpdate(name=job.name)
+        )
+
+        project_inputs = await webserver_api.get_project_inputs(project_id=project.uuid)
+
+        file_param_nodes = {}
+        for node_id, node in project.workbench.items():
+            assert node.outputs is not None  # nosec
```

Comment:
how are you sure this is always the case? I would drag this in the if condition or raise and handle an error in that situation

Reply:
I removed this code for another PR, hence resolving this conversation